### PR TITLE
Aphy plushie

### PR DESCRIPTION
**Pull request by listerla**
Adds an IPC plushie, with lore and approval straight from Nienna. Sprite taken from Paradise.

image
image

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1005,6 +1005,13 @@
 	icon_state = "xanaplushie"
 	phrase = "warble!"
 
+/obj/item/toy/plushie/ipc
+	name = "Aphy plushie"
+	desc = "A plushie of an old Hephaestus mascot, Aphy."
+	desc_fluff = "Aphy, a play on the name Aphrodite, was Hephaestus Industries' first baseline prototype. While the original Aphy is on display in Hephaestus' Mars headquarters, the unit has become a cutesy mascot in recent years."
+	icon_state = "ipcplushie"
+	phrase ="Bwoop!"
```
Comment (Geevies) [inline code/game/objects/items/toys.dm:1013]:
```suggestion
	phrase = "Bwoop!"
```

**Review by Geevies — APPROVED** [review]

**Comment by listerla** [comment]
!review

**Comment by Spacemanspark** [comment]
@Drakeven is the artist who made the sprite.

**Comment by listerla** [comment]
I believe the artist was actually @DogsParadise

**Comment by DogsParadise** [comment]
Wait how did you get my sprite?

**Comment by listerla** [comment]
From the Paradise repo, IIRC? I didn't collect it myself. If you'd prefer it not be used, I can close this PR.

**Comment by DogsParadise** [comment]
Nah it's fine. Was just super confused

**Comment by Drakeven** [comment]
I made a different sprite!

**Review by Casper3667 — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Pull request merged**